src/View/User/Includes/_course_white_block.html.twig

```diff
@@ -11,7 +11,7 @@
             </p>
         </div>
         <div class='carousel'>
-            {% include 'User/Includes/_carousel.html.twig' %}
+            {% include 'User/Includes/_carousel.html.twig' with {image1:'faceharas.jpg', image2:'poneycarousel.jpg', image3:'prairieharas.jpg'} %}
```

Comment:
same comment as on _carousel.html.twig

Reply:
yes but the response is on comment on _carousel.html.twig